### DISCUSSION TITLE
I'm having trouble installing the stripe app on my dashboard. Please help me!

### DISCUSSION THREAD

**Discussion by Kinghood15** [saleor/apps#1038] [Q&A]
Dashboard version 3.16
Saleor Core version 3.16
Error 
![image](https://github.com/saleor/apps/assets/74972851/2430858d-151f-4d9c-b0cf-110ccf1f04c9)
and i check api error 
![image](https://github.com/saleor/apps/assets/74972851/03f2d504-f61d-4734-ab2f-63df2425bd2d)
Can we use an external app without the Saleor cloud?

**Comment by krzysztofwolski** [comment]
> Can we use an external app without the Saleor cloud?

External Apps can be used with self-hosted Saleor instances.

> Could not save the auth data

Probably, the issue is related to APL: https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-sdk/apl

Is your application using your own APL or one of the implementations from the App SDK? 

To debug this you'll need to check app logs. Here are links that should help:
- AppSDK logging https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-sdk/debugging
- App logging (`APP_LOG_LEVEL`) https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-examples#environment-variables
- for additional debugging of the registration handler you can use hooks https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-sdk/api-handlers
